core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -55,7 +55,6 @@
             class="nav-item  oppia-navbar-clickable-dropdown e2e-test-home-oppia-list-item">
           <a (keydown)="onMenuKeypress($event, 'homeMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_OPEN})"
              class="nav-link oppia-navbar-tab"
-             role="menuitem"
              aria-haspopup="true"
```

Comment:
Should a aria-label also be added here and elsewhere?

Reply:
Sure, @Rijuta-s  can you please suggest an aria-label that will be suitable here?

Reply:
@Rijuta-s I have added the relevant aria-labels. Thanks